sdk/keyvault/keyvault-keys/src/core/keyVaultClient.ts

```diff
@@ -5266,7 +5263,7 @@ const createKeyOperationSpec: coreHttp.OperationSpec = {
   path: "keys/{key-name}/create",
   urlParameters: [Parameters.vaultBaseUrl, Parameters.keyName0],
   queryParameters: [Parameters.apiVersion],
-  headerParameters: [Parameters.acceptLanguage],
```

Comment:
This file needs few more additions in getDeletedKeysOperationSpec, importKeyOperationSpec, etc. Else, the span will not be serialized to the service.

Reply:
Done with https://github.com/Azure/azure-sdk-for-js/pull/4654/commits/e2c529fb46463ec26bb7daf7ec63905c74450f30

Reply:
We need to speak with @daviwil and @schaabs to make sure that we can apply these changes to a future version of our core files, since they're auto-generated.